protoc-gen-docs/htmlGenerator.go

```diff
@@ -366,7 +365,11 @@ func (g *htmlGenerator) generateFileHeader(top *fileDescriptor, numEntries int)
 		if top == nil {
 			croak("PANIC: null file %v", name)
```

Comment:
Can this just be a warning instead of terminating the process? See the warn method...